packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -430,7 +430,7 @@ class DebuggerScreen extends Screen {
       breakpointsView.element,
       PNavMenuItem('Scripts')
         ..add(
-          scriptCountDiv = span(text: '0', c: 'counter'),
+          scriptCountDiv = span(text: 'Unavailable', c: 'counter'),
```

Comment:
I think we generally want the badge decorations to be small. Perhaps leave this as `0`, and change to something else (`-`) is getScripts() is not available? If we want to explain further (which may or may not be necessary), we could switch out the div content of the scripts pane w/ the text `Not available in Dart xxx`.

Reply:
Changes to '-'. Changing the content was a little more complicated (since we don't know until "later" whether they're supported, but having Unavailable (or `-`) is unspecific enough that it's fine to show even if it is supported, until we get them. Since this seems like an edge case (I think most people will run with a newer version) I didn't think it was worthwhile, but if you think it might happen much and/or is confusing without, let me know and I'll sort it :-)

Reply:
👍 sounds great

Reply:
I'm not sure if that means sounds great just leaving as `-`, or looking at adding a note in the content :)

Reply:
Sorry - I was referring the to the top part of the note :) Using just `-` as the badge content sounds great. If getScripts() is supported it'll be replaced with the script count (and because it's about the same size as the script count, users won't see a big change in the component size once the content loads).

Reply:
Gotcha! 👍